Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.14-alpine as build
+COPY . /s5cmd/
+RUN cd /s5cmd/ && \
+    go build -mod=vendor -o s5cmd main.go
```

Comment:
hmm, binary is missing the correct version information. Let's fix it in this PR.

- We have a `build` target in Makefile. It's better to use it instead of plain `go build`. 
- That target doesn't embed the correct tag and git commit information.

```
VERSION := `git describe --abbrev=0 --tags || echo "0.0.0"`
BUILD := `git rev-parse --short HEAD`
LDFLAGS=-ldflags "-X=github.com/peak/s5cmd/version.Version=$(VERSION) -X=github.com/peak/s5cmd/version.GitCommit=$(BUILD)"

.PHONY: build
build:
	@go build ${GCFLAGS} ${LDFLAGS} -mod=vendor .
```
should do the job.

Reply:
thanks, done.

Reply:
🚀